Libraries/Components/Navigation/NavigatorIOS.ios.js

```diff
@@ -233,6 +266,11 @@ var NavigatorIOS = React.createClass({
       rightButtonTitle: PropTypes.string,
 
       /**
+       * If set, the right header button will appear with this system icon
+       */
+      rightButtonSystemItem: PropTypes.oneOf(Object.keys(SystemItems)),
```

Comment:
Should I make the change from item to icon just for the public API or also for the internal Obj C stuff?